patient_case_summary/patient_case_summary.py

```
@@ -0,0 +1,788 @@
+    StopEvent,
+    Context,
+    Workflow,
+    step,
+)
+from llama_index.core.llms import LLM
+from typing import Optional, Tuple
+from pydantic import BaseModel
+from llama_index.core import SimpleDirectoryReader
+from llama_index.core.schema import Document
+from llama_index.core.agent import FunctionCallingAgentWorker
+from llama_index.core.prompts import ChatPromptTemplate
```

Ruff (0.8.2)

Redefinition of unused `ChatPromptTemplate` from line 236

Remove definition: `ChatPromptTemplate`

(F811)